apis/swagger.yml

```diff
@@ -371,6 +371,34 @@ paths:
         500:
           $ref: "#/responses/500ErrorResponse"
 
+  /images/prune:
+    post:
+      summary: "Delete unused images"
+      produces:
+        - "application/json"
+      operationId: "ImagePrune"
+      parameters:
+        - name: "filters"
+          in: "query"
+          description: |
+            Filters to process on the prune list, encoded as JSON (a `map[string][]string`). Available filters:
+
+            - `dangling=<boolean>` When set to `true` (or `1`), prune only
```

Comment:
Do we support this `dangling` filter now? If currently not, maybe you should change this.

Reply:
@hellolijj I think we should also add dangling filter, so that users can see what will be removed before remove it. wdyt?

Reply:
I asked @fuweid  about this requirement. He said there was no need for `hangling` filter. right? @fuweid